sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/LocalRelation.scala

```diff
@@ -43,11 +43,10 @@ object LocalRelation {
   }
 }
 
-case class LocalRelation(
-    output: Seq[Attribute],
-    data: Seq[InternalRow] = Nil,
-    // Indicates whether this relation has data from a streaming source.
-    override val isStreaming: Boolean = false)
+case class LocalRelation(output: Seq[Attribute],
```

Comment:
although we should not include this style change in the original commit, since it's already there, let's not bother about reverting it back.